BackwardMacros.cmake

```diff
@@ -74,17 +77,20 @@ if (${STACK_DETAILS_AUTO_DETECT})
 		set(STACK_DETAILS_DW TRUE)
 		set(STACK_DETAILS_BFD FALSE)
 		set(STACK_DETAILS_BACKTRACE_SYMBOL FALSE)
+		set(BACKWARD_USED_LIBRARY "#define BACKWARD_HAS_DW 1")
```

Comment:
No, as this is the final string, that gets inserted into the backward_config.hpp